include/hpp/constraints/differentiable-function.hh

```diff
@@ -188,6 +188,17 @@ namespace hpp {
 
       bool operator== (DifferentiableFunction const & other) const;
       bool operator!= (DifferentiableFunction const & b) const;
+
+      /// Return pair of joints whose pose modifies the value if any
```

Comment:
Can you be more accurate:
Return pair of joints the relative pose between which modifies the function value if any